lib/core/include/irods/irods_logger.hpp

```diff
@@ -43,6 +43,7 @@ namespace irods::experimental
 
         enum class level
         {
+            sql,
```

Comment:
What is the motivation behind these changes to the logging API?

Reply:
The sql loglevel is distinct in the old log api, I would guess because it's even more prone to filling the logs than trace.

Reply:
The log levels are pretty standard, so we don't want to introduce one people are unfamiliar with.

A better approach would be to introduce a new category.